cmd/stack_removefromrepo.go

```diff
@@ -121,25 +121,15 @@ The updated repository index file is created in  ~/.appsody/stacks/dev.local dir
 					if err != nil {
 						return errors.Errorf("Error checking status of %s", localIndexFile)
 					}
-					if exists && useLocalCache {
-						log.Debug.Log(localIndexFile, " exists in the appsody directory and use-local-cache is true")
+					if exists {
+						log.Debug.Log(localIndexFile, " exists in the appsody directory")
 					} else {
 						log.Info.Log("Repository index file not found - unable to remove stack")
 						return nil
 					}
 				}
 			} else {
-				log.Debug.Log(repoName, " does not exist within the repository list")
-				exists, err := Exists(localIndexFile)
-				if err != nil {
-					return errors.Errorf("Error checking status of %s", localIndexFile)
-				}
-				if exists && useLocalCache {
-					log.Debug.Log(localIndexFile, " exists in the appsody directory and use-local-cache is true")
-				} else {
-					log.Info.Log("Repository index file not found - unable to remove stack")
-					return nil
-				}
+				return errors.Errorf("%v does not exist within the repository list", repoName)
```

Comment:
Would this also need to be changed to printing a debug/info and returning a `nil`? Or is it acceptable to keep it as an error if you're explicitly trying to remove a stack that isn't found?